x-pack/plugins/secops/public/components/timeline/events.ts

```diff
@@ -29,7 +28,7 @@ export type OnToggleDataProviderNegated = (
 ) => void;
 
 /** Invoked when a user selects a new minimap time range */
-export type OnRangeSelected = (range: Range) => void;
+export type OnRangeSelected = (range: string) => void;
```

Comment:
This is another example where we lose type safety

Reply:
It's the same area of code. It's that Range is based on user facing text for types as far as I understand.

Reply:
`Range` was declared (before it's removal in this PR) as:

`export const Ranges: Range[] = ['1 Day', '1 Week', '1 Month', '1 Year']`

which defined the set of all valid ranges. Most public APIs used this type to ensure only valid values (or in the case of the Redux store, valid states) could be be represented. This code is going away soon, but it's notable that the i18n resulted in a loss of type safety.